sql/src/Main.sk

```diff
@@ -653,6 +681,7 @@ fun execReplicationId(args: Cli.ParseResults, options: SKDB.Options): void {
 }
 
 fun execWriteCsv(args: Cli.ParseResults, options: SKDB.Options): void {
+  SKDBMonitor.setName("skdb.write.csv");
   ensureContext(args);
```

Comment:
Do we set the user in the monitoring for write csv?

Reply:
Now, no.
This is one of the questions I ask myself in terms of safety, but I don't really have an answer.
You certainly have more experience than I do on this subject.

Reply:
I don't think it hurts to log who the user is. Obviously logging the data is much riskier and we shouldn't do that. But knowing the user could really help with figuring out errors. And in terms of safety we probably really want to know who is interacting with databases.